services/worker-manager/src/api.js

```diff
@@ -383,7 +383,7 @@ builder.declare({
     limit: parseInt(req.query.limit || 100, 10),
     matchPartition: 'exact',
   };
-
+  
```

Comment:
Extra whitespace (and at one point below).  I expect the lint check will complain about this.  You can run lint locally with `yarn lint`.